generator-elide/generators/app/templates/src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE hibernate-configuration PUBLIC
+        "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+        "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+
+<hibernate-configuration>
+    <session-factory>
+        <!-- Database connection settings -->
+        <property name="connection.driver_class">com.mysql.jdbc.Driver</property>
```

Comment:
I wonder if this driver should be configurable with a default of `com.mysql.cj.jdbc.Driver`?